src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,88 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.command.Command;
+import duke.command.CommandCheck;
+import duke.command.CommandDeadline;
+import duke.command.CommandDelete;
+import duke.command.CommandDone;
+import duke.command.CommandEvent;
+import duke.command.CommandFind;
+import duke.command.CommandHelp;
+import duke.command.CommandList;
+import duke.command.CommandTodo;
+
+/**
+ * Parser is the class that handles the parsing of strings.
+ *
+ * @author Loh Wen Hao Aaron
+ *
+ */
+public class Parser {
+    private static final String[] TASK_TYPES = {"T", "D", "E"};
+    private static ArrayList<String> storedInput;
+
+    public Parser() {
+    }
+
+    /**
+     * Takes in a string and checks if it corresponds to a task type.
+     * If it does, return its index number in the TASK_TYPES array.
+     * If it does not, return -1.
+     *
+     * @param s  String to be checked.
```

Comment:
Is there an additional whitespace here?
```suggestion
     * @param s String to be checked.
```